src/tlo/methods/healthsystem.py

```diff
@@ -2697,6 +2824,22 @@ def record_never_ran_hsi_event(self,
             self._never_ran_appts[appt_type] += number
             self._never_ran_appts_by_level[level][appt_type] += number
 
+    def record_hsi_event_no_blank_appt_footprints(self,
+                                                  treatment_id: str,
+                                                  hsi_event_name: str,
+                                                  appt_footprint: Counter,
+                                                  level: str
+                                                  ) -> None:
+        """Add information about a`HSI_Event` that has a non-blank appointment footprint to the running summaries."""
```

Comment:
Similar to my comment above. This docstring also needs to include descriptions for each of the parameters
```suggestion
        """Add information about a`HSI_Event` that has a non-blank appointment footprint to the running summaries.
        :param treatment_id: .....
        :param hsi_event_name: .....
        :param appt_footprint: .....
        :param level: ....."""
```